### DISCUSSION TITLE
Specifying a specific parametrization of a test

### DISCUSSION THREAD

**Discussion by 1p1e3** [pytest-dev/pytest#12740] [Q&A] [answered]
hello teachers，i'm a beginner，i'm reading the documentation for pytest，i'm learning how to invoke pytest，
i have no idea about "Specifying a specific parametrization of a test"，i don't know how to use it，can you provide me with a simple usage example? thank you very much！🫡🫡🫡

![1](https://github.com/user-attachments/assets/257e7cbe-2359-4fc1-bf19-e2142c158c65)

**Comment by The-Compiler** [comment, marked as answer]
You might want to read more about [parametrizing test functions](https://docs.pytest.org/en/stable/how-to/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions) in order to understand that example.

Based on that, imagine a `test_mod.py` like

```python
import pytest

@pytest.mark.parametrize("a, b", [("x1", "y2"), ("v3", "w4")])
def test_func(a, b):
    ...
```

which will result in two test cases:

```
test_mod.py::test_func[x1-y2]
test_mod.py::test_func[v3-w4]
````

and the shown invocation will run the first one only.
